pkg/controller/master/node/promote_controller.go

```diff
@@ -44,10 +45,9 @@ const (
 	HarvesterMgmtNodeLabelKey    = HarvesterNodeRoleLabelPrefix + "management"
 	HarvesterWorkerNodeLabelKey  = HarvesterNodeRoleLabelPrefix + "worker"
 
-	HarvesterLabelAnnotationPrefix      = "harvesterhci.io/"
-	HarvesterManagedNodeLabelKey        = HarvesterLabelAnnotationPrefix + "managed"
-	HarvesterPromoteNodeLabelKey        = HarvesterLabelAnnotationPrefix + "promote-node"
-	HarvesterPromoteStatusAnnotationKey = HarvesterLabelAnnotationPrefix + "promote-status"
+	HarvesterManagedNodeLabelKey        = util.HarvesterManagedNodeLabelKey
+	HarvesterPromoteNodeLabelKey        = util.HarvesterPromoteNodeLabelKey
+	HarvesterPromoteStatusAnnotationKey = util.HarvesterPromoteStatusAnnotationKey
```

Comment:
I don't see a point in redefining these constants. Can't we just use the `util.*` constants?